#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-shellrun.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Don't include broken and unused comment.

@DPATCH@

diff -Naurp autorun4linuxcd-0.12.orig/shellrun/shellrun.c autorun4linuxcd-0.12/shellrun/shellrun.c
--- autorun4linuxcd-0.12.orig/shellrun/shellrun.c	2007-04-02 10:39:18.000000000 +0000
+++ autorun4linuxcd-0.12/shellrun/shellrun.c	2008-08-09 18:21:08.000000000 +0000
@@ -16,7 +16,7 @@ open=shellrun.exe index.html
 
 #include <windows.h>
 
-statuc char[] RCS_ID = "$Id: shellrun.c,v 1.1 2004/06/13 11:05:04 jv Exp $ ";
+//statuc char[] RCS_ID = "$Id: shellrun.c,v 1.1 2004/06/13 11:05:04 jv Exp $ ";
    
 int WINAPI WinMain( HINSTANCE hInstance, 
 		    HINSTANCE hPrevInstance, 
